app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -215,6 +211,16 @@ def finish(*_args)
 
   alias_method :abort_job, :cleanup
 
+  def cancel(*args)
+    _log.info "Job Canceling"
+    if self.state != "canceling" # ensure change of states
+      signal :cancel
+    else
+      unqueue_all_signals
+      queue_signal(:abort_job, "Job Canceled", "cancel", *args)
```

Comment:
Also once the status 'cancel' is set the job does not show up in the tasks screen. I'm not sure what the expected behavior is but it seems to me that canceled jobs show up as ok.

@roliveri What should be the status on canceled scan jobs? do they show up in the tasks screen